### PR TITLE
Oop mini project

### PR DESCRIPTION
**Pull request by ACNtam**
Hi Geoff, 
Status
Ready | completed mini project

Description
Please look over my HTML and JS code and let me if any changes need to be made.

Thank you

### CODE REVIEW

File: oop-mini-project/event.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+eventObj1.addAvailableTickets("vampire", 99);
+
+eventObj2.addAvailableTickets("General Admission", 25)
+eventObj2.addAvailableTickets("Floor Seating", 80)
+
+eventObj3.addAvailableTickets("Orchestra", 300)
+eventObj3.addAvailableTickets("Mezzanine", 200)
+eventObj3.addAvailableTickets("Balcony", 100)
+
+eventObj1.allTickets();
```
Comment (Geoff-at-iterable) [inline oop-mini-project/event.js:100]: Do we want to wrap this in a console.log()?

File: portfolio-website/.gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
```
Comment (Geoff-at-iterable) [inline portfolio-website/.gitignore]: It looks like the portfolio website files are part of this git commit. I would delete the portfolio-website folder on this branch before merging this in.

File: .gitignore [hunk 3]
```diff
@@ -1 +0,0 @@
-.DS_Store
```
Comment (Geoff-at-iterable) [inline .gitignore]: I would add this back.

**Comment by Yosolita1978** [comment]
Hi Alice! 
Good first step with your OOP. 

You already create the method `searchTickets`, but you're not using it. Try to see what happens when you do `console.log(eventObj1.searchTickets(0, 200));` 

And the same with the TicketType class. Try to create instances of that class. Can you think of a way of doing it? How can you use the class that you already have defined in line 82

**Review by Geoff-at-iterable — APPROVED** [review]

**Pull request merged**